src/generic-container/generic-container.ts

```diff
@@ -58,7 +59,7 @@ export class GenericContainer implements TestContainer {
   protected tmpFs: TmpFs = {};
   protected healthCheck?: HealthCheck;
   protected waitStrategy?: WaitStrategy;
-  protected startupTimeout = 60_000;
+  protected startupTimeout = EnvConfig.defaultSetupTimeout();
```

Comment:
this to me reads as startupTimeout = the default, perhaps it should be `getStartupTimeout` ?